shell/platform/android/android_context_gl.cc

```diff
@@ -112,43 +118,86 @@ static bool TeardownSurface(EGLDisplay display, EGLSurface surface) {
 }
 
 // For onscreen rendering.
-static EGLResult<EGLSurface> CreateWindowSurface(
-    EGLDisplay display,
-    EGLConfig config,
-    AndroidNativeWindow::Handle window_handle) {
+bool AndroidContextGL::CreateWindowSurface(
+    ftl::RefPtr<AndroidNativeWindow> window,
+    bool* srgb_support) {
   // The configurations are only required when dealing with extensions or VG.
   // We do neither.
+
+  window_ = std::move(window);
+  EGLDisplay display = environment_->Display();
+
+  EGLint srgb_attributes[] = {
+      EGL_GL_COLORSPACE_KHR, EGL_GL_COLORSPACE_SRGB_KHR,
```

Comment:
I think it should fail - at least it did when I (wrongly) tried passing these extensions to eglChooseConfig.

But I think what you suggest is much cleaner.  Will do that.